src/main/java/com/javahelps/restservice/CodeReview.java

```diff
@@ -1 +1,42 @@
+import java.time.LocalDate;
+import java.time.temporal.ChronoUnit;
+import java.util.Locale;
 
+public class CodeReview {
+
+    Integer x;
```

Comment:
please declare a good name instead of using x like for example ; age, years , make it more clear  and indicate what it used for